TicTacToe2/TicTacToe/Player.cs

```diff
@@ -6,30 +6,25 @@ namespace TicTacToe2.TicTacToe
 {
     class Player
     {
+        public int row;
+        public int col;
+
+        public char Symbol { get; }
 
         public Player(char symbol)
         {
             Symbol = symbol;
         }
 
-        public void GetMove(out int x, out int y)  //we need to return multiple values
+        public void GetMove(out int x, out int y)
```

Comment:
This function should contain all of the logic for getting a move. Any and all logic used to get the x y coordinates of a move from the player should go here. No other class should know anything about how this process is done. The logic should be encapsulated in this class.